index.html

```diff
@@ -5838,6 +5838,26 @@ <h3>Group Control</h3>
         </p>
       </section>
     </section>
+      <section>
+      <h2>Changing the DID controller</h2>
+      <p>
+The <a>DID controller</a> for <a>DID document</a> may change over time. However,
+depending on how it is implemented, a change in the <a>DID controller</a> may
+not be apparent from changes to the <a>DID document</a> itself. For example, if
+the change is implemented through a shift in ownership of the underlying
```

Comment:
```suggestion
not be made apparent by changes to the <a>DID document</a> itself. For example,
if the change is implemented through a shift in ownership of the underlying
```

Reply:
I *think* this change happened through another pathway, but I'm not certain, and it appears un-accepted here.  Please check/confirm?